libpolys/polys/kbuckets.cc

```diff
@@ -13,7 +13,7 @@
 //#include <kernel/pShallowCopyDelete.h>
 #include <coeffs/coeffs.h>
 #include <polys/monomials/ring.h>
-//#include <kernel/p_Procs.h>
```

Comment:
This has no effect and libpolys/polys/templates/p_Procs.h must not be used outside of
the template stuff!